demo/gallery/vat-alice.js

```diff
@@ -216,16 +216,34 @@ function makeAliceMaker(E, log) {
           // willing to offer for it
           // sellToGallery returns an invite to the smart contract
           const { inviteP, host } = await E(gallery).sellToGallery(amount);
-          const seatP = E(host).redeem(inviteP);
-          E(seatP).offer(exclusivePixelPaymentP);
-          const dustPurseP = E(dustIssuer).makeEmptyPurse();
-          const pixelPurseP = E(pixelIssuer).makeEmptyPurse();
-          return E(gallery).collectFromGallery(
+          const seatP = await E(host).redeem(inviteP);
+          await E(seatP).offer(exclusivePixelPaymentP);
```

Comment:
As I understand it, the await on 219 means that seatP should be 'seat', and you don't need E(seatP).